mu4/scenes/notation/view/notationviewinputcontroller.cpp

```diff
@@ -163,6 +166,75 @@ void NotationViewInputController::hoverMoveEvent(QHoverEvent* ev)
     }
 }
 
+void NotationViewInputController::dragEnterEvent(QDragEnterEvent* ev)
+{
+    //LOGI() << "ev: " << ev;
+
+    const QMimeData* dta = ev->mimeData();
```

Comment:
Do you need nullptr checks here?

Reply:
I don't know, nothing wrote about this in the documentation. I added check.